app/Http/Controllers/ApiController.php

```diff
@@ -58,11 +58,6 @@ public function update(Request $request)
             'error_msg'     => 'nullable|string',
             'limitsw_open'   => 'nullable|boolean',
             'limitsw_closed' => 'nullable|boolean',
```

Comment:
I dont send limit switch data at all to the server.
Not even sending that as sensor data.

Reply:
Fixed 898bba8